admin/cipipeline/phase2.md

```diff
@@ -0,0 +1,45 @@
+# CI/CD Pipeline Phase 2
+
+## Pipeline graph
+
+![Drawio Pipeline Graph](phase2.drawio.png)
+
+## Currently Functional:
+
+### Prettier for Code Linting
+
+We are going to have members apply the `prettier` extension/package to every file that they edit for the project in order to maintain consistent code styling.
+
+Prettier is set up to auto-format any file that is being saved, ensuring that the code practices are uniform for all members
+
+### Codacy for Code Quality
+
+We are going to have the Codacy app run quality tests on code to ensure that the code will be maintainable and follows best practices on each pull request.
+
+### Testing:
+
+#### Jest
+
+We will be using Jest in order to implement our unit tests. The `jest` package needs to be installed in order to use Jest properly to run tests. You can use the `npm run test` command to run the unit tests. The unit tests can be found in the `__tests__/unit` directory.
+
+#### WebDriverIO
+
+We will be using WebDriverIO in order to complete end-to-end testing within the built Electron package. The `wdio` packages should be installed and you must build the app using one of the `npm run package` commands that can be found in `package.json` in order to make this work. Use the `npm run wdio` command in order to run the end-to-end tests. The tests themselves can be found in the `__tests__/e2e` directory.
```

Comment:
We will be using WebDriverIO in order to complete end-to-end testing within the built Electron package. The `wdio` packages should be installed and you must build the app using one of the `npm run package` commands that can be found in `package.json` in order to make this work. Use the `npm run wdio` command in order to run the end-to-end tests. The tests themselves can be found in the `__tests__/e2e` directory.

We run these less frequently, usually before merging and requesting reviews for PRs.